src/components/modals/send-modal/index.js

```diff
@@ -234,9 +240,20 @@ const SendModal = ({
                     onChange={feeRateOnChange}
                   />
                 </InputGroup>
+                { boostRequired ?
+                  <InputGroup className="mb-3">
+                    <Form.Label>Select an output value</Form.Label>
+                    <Form.Range
+                      min="330"
+                      max="10000"
+                      defaultValue={boostOutputValue}
```

Comment:
Is better if you remove defaultValue and use `value={boostOutputValue}`

Reply:
thanks